src/test/java/org/nuxeo/binary/metadata/test/BaseBinaryMetadataTest.java

```diff
@@ -0,0 +1,40 @@
+package org.nuxeo.binary.metadata.test;
+
+import static org.junit.Assert.assertFalse;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Optional;
+
+import javax.inject.Inject;
+
+import org.nuxeo.binary.metadata.api.BinaryMetadataService;
+import org.nuxeo.ecm.core.api.CoreSession;
+import org.nuxeo.ecm.platform.commandline.executor.api.CmdParameters;
+import org.nuxeo.ecm.platform.commandline.executor.api.CommandLineExecutorService;
+
+/**
+ * @since 11.1
+ */
+public abstract class BaseBinaryMetadataTest {
+    @Inject
+    protected CoreSession session;
+
+    @Inject
+    protected BinaryMetadataService binaryMetadataService;
+
+    @Inject
+    protected CommandLineExecutorService commandLineExecutorService;
+
+    protected void itShouldFailIfOriginalFileIsCreatedByExifTool() throws IOException {
+        CmdParameters defaultCmdParameters = commandLineExecutorService.getDefaultCmdParameters();
+        Path tmpDirectory = Paths.get(defaultCmdParameters.getParameter("nuxeo.tmp.dir"));
+
+        Optional<Path> optional = Files.find(tmpDirectory, 1, (p, bf) -> p.toFile().getName().endsWith("_original"))
```

Comment:
The Javadoc states that the `Files#find` method must be used in a try-with-resources statement.